src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,180 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
Don't begin class or method implementation with an empty line. There should not be an empty line after the class declaration (line 5).